contracts/earn/strategies/TraderJoe/StrategyTraderJoeDualLP.sol

```diff
@@ -0,0 +1,197 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.12;
+
+import '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import '@openzeppelin/contracts/token/ERC20/SafeERC20.sol';
+import '@openzeppelin/contracts/math/SafeMath.sol';
+import '@openzeppelin/contracts/access/Ownable.sol';
+import '@openzeppelin/contracts/utils/Pausable.sol';
+import '@uniswap/lib/contracts/libraries/TransferHelper.sol';
+
+import '../../../interfaces/PancakeSwap/IMasterChef.sol';
```

Comment:
Already declared in BaseStrategy, therefore using `PancakeSwap/IMasterChef.sol` here, otherwise will compile failed.